src/Tribe/Customizer.php

```diff
@@ -583,11 +583,13 @@ public function register( WP_Customize_Manager $customizer ) {
 			 * Allows people to Register and de-register the method to register more Fields
 			 *
 			 * @since 4.4
+			 * @since TBD Add Customizer instance as a parameter.
 			 *
 			 * @param array                $section
 			 * @param WP_Customize_Manager $manager
+			 * @param Tribe__Customizer    $customizer The current customizer instance.
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :warning: Line exceeds 80 characters; contains 87 characters

<sub>hash: 3ab3ce3a75a6a333a078fdba06d26505</sub>